Jenkinsfile_nightly

```diff
@@ -42,8 +42,24 @@ withNightlyPipeline(type, product, component) {
   env.TEST_URL = params.URL_TO_TEST
 
   after('checkout') {
-    sh 'curl -L https://github.com/git-lfs/git-lfs/releases/download/v2.7.1/git-lfs-linux-amd64-v2.7.1.tar.gz | tar xz -C ./bin git-lfs'
-    sh 'PATH=$PATH:./bin git lfs install --local && PATH=$PATH:./bin git lfs pull'
+      sh '''
+        set -e
+    
+        echo "7be85238cbbb957ab25de52b60279d40ba40d3faa72eeb2cb9fa77d6d92381e5  git-lfs-v2.7.1.tar.gz" > git-lfs-v2.7.1.checksum.txt
+        curl -L https://github.com/git-lfs/git-lfs/releases/download/v2.7.1/git-lfs-linux-amd64-v2.7.1.tar.gz > git-lfs-v2.7.1.tar.gz
+        sha256sum -c git-lfs-v2.7.1.checksum.txt
+        
+        tar xz -C bin/ git-lfs -f git-lfs-v2.7.1.tar.gz
+        ls -lash bin/
+      '''
+
+      sh '''
```

Comment:
I do not see a reason to have two bash scripts. I suggest combining these together as they are both aim to achieve common goal.

Reply:
👍